mixer/pkg/config/crd/store.go

```diff
@@ -146,12 +150,26 @@ func (s *Store) Init(kinds []string) error {
 	s.informers = make(map[string]cache.SharedInformer, len(kinds))
 	remaining := s.checkAndCreateCaches(d, lwBuilder, kinds)
 	if len(remaining) > 0 {
+		for _, k := range remaining {
+			if s.isCriticalKind(k) {
```

Comment:
Please add a configurable timeout and an active poll for missing critical kinds, giving up so early is actually expensive. Default waiting for upto 10s is reasonable.

Reply:
Done. Use the retryTimeout and retryInterval that are already existed. tetryTimeout is configurable through URL query param.